.github/PULL_REQUEST_TEMPLATE/development.md

```diff
@@ -0,0 +1,42 @@
+## Pre-Review Checklist for the PR Author
+
+1. [ ] Branch follows the naming format (`iox-#123-this-is-a-branch`)
+1. [ ] Commits messages are according to this [guideline][commit-guidelines]
+    - [ ] Commit messages have the issue ID (`iox-#123 commit text`)
+    - [ ] Commit messages are signed (`git commit -s`)
+    - [ ] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
+1. [ ] Code according to our coding style and naming conventions
+1. [ ] Unit tests have been written for new behavior
+1. [ ] Public API changes are documented via doxygen
+1. [ ] Copyright owner are updated in the changed files
+1. [ ] Update the PR title
+   - Follow the same conventions as for commit messages
+   - Link to the relevant issue
+1. [ ] Relevant issues are linked
+1. [ ] Add sensible notes for the reviewer
+1. [ ] All checks have passed
+1. [ ] Assign PR to reviewer
+
+[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+[eca]: http://www.eclipse.org/legal/ECA.php
+
+## Notes for Reviewer
```

Comment:
Do the notes also include a brief description of the changes? Or is mentioned somewhere else?

Reply:
yes, I would assume so